### DISCUSSION TITLE
Clock-like display of items - sort, list, color items based on a filter e.g.: time of creation, last access, etc.

### DISCUSSION THREAD

**Discussion by claudiusraphael** [Schneegans/Fly-Pie#320] [Ideas & Feature Discussions]
## The Motivation
I wonder if the actual structure of your code allows to configure the specific orientation/degree of items. If so a clock-like display would allow to embed another context, namely time and date.

## The Solution
Let's say a user has a regular line-up of apps, which are started at specific times a day - these could be represented as such - placed where the dial/index would point to on a clock.

A Folder containing subfolders could be sorted by creation or access time/date and displayed accordingly in the menu, like on a clock.

## The Alternatives
Since you are already at it, lol .. how about Fly-Pie GUI as a standalone app cross-platform with a service/daemon that binds to the given OS hooks/signals/events and yeah, with this clock-mode-idea included and the option to constantly display Fly-Pie, but when not in focus showing an actual clock ... (Ok, ok, yayaa ... one may dream, right?)

**Comment by Schneegans** [comment]
Hi there! Thank you very much for the suggestion! With the "fixed angle" property of the items you can already specify the direction of each item. This works for sub-menus and actions; you cannot specify the direction of individual items of predefined sub-menus such as the bookmarks or recent files.

So you could use this feature already to manually create menus where the items are placed according to the time of day at which you usually use them.

Positioning the items automatically based on access or creation time would be possible implementation-wise. However, I think that it is not that useful under most circumstances. Pie-menus work best if item positions change as little as possible.

I am currently working on [Kando](https://github.com/kando-menu/kando), a cross-platform successor to Fly-Pie. This will also support manually specifying item directions!
